sonic-thermalctld/scripts/thermalctld

```diff
@@ -463,13 +463,22 @@ class TemperatureUpdater(logger.Logger):
         state_db = daemon_base.db_connect("STATE_DB")
         self.table = swsscommon.Table(state_db, TemperatureUpdater.TEMPER_INFO_TABLE_NAME)
 
+        self.is_chassis_system = chassis.is_modular_chassis()
+        if self.is_chassis_system:
+            my_slot = chassis.get_my_slot()
+            table_name = TemperatureUpdater.TEMPER_INFO_TABLE_NAME+'_'+str(my_slot)
+            global_state_db = daemon_base.db_connect("GLOBAL_STATE_DB")
```

Comment:
Looks like we need to change here too to CHASSIS_STATE_DB

Reply:
The 2nd commit here already has fixed this. Please review.